llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1446,7 +1446,8 @@ static void transformRecipestoEVLRecipes(VPlan &Plan, VPValue &EVL) {
   VPTypeAnalysis TypeInfo(CanonicalIVType);
   LLVMContext &Ctx = CanonicalIVType->getContext();
   SmallVector<VPValue *> HeaderMasks = collectAllHeaderMasks(Plan);
-
+  VPValue *AllOneMask =
```

Comment:
Also use for select VPInstruction below?

Reply:
A separate NFC patch would be better :)

Reply:
Sorry, I don't understand what you mean, the vp.select does not have a mask parameter. `declare <16 x i32>  @llvm.vp.select.v16i32 (<16 x i1> <condition>, <16 x i32> <on_true>, <16 x i32> <on_false>, i32 <evl>)`

Reply:
I think Florian means vp.merge.

Reply:
I will create a new patch to deal with it.